src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx

```diff
@@ -39,11 +37,7 @@ const VenueWizardCreate: React.FC<VenueWizardCreateProps> = ({
     );
   }
 
-  return (
-    <AdminRestricted>
-      <Details previous={previous} dispatch={dispatch} data={state} />
-    </AdminRestricted>
-  );
```

Comment:
Suggesting we leave this component in place. Remove the secondary (visual) effect of `AdminRestricted` itself as per https://doesitsparkle.slack.com/archives/C028JG5TUMC/p1632471222090800?thread_ts=1632466598.085100&cid=C028JG5TUMC